tests/experimental/datasources/test_schemas.py

```diff
@@ -51,6 +51,8 @@ def test_vcs_schemas_match(zep_ds_or_asset_model: Type[pydantic.BaseModel]):
     if schema_path.name in (
         "SqliteDatasource.json",
         "SqliteTableAsset.json",
+        "SASAsset.json",
+        "PandasDatasource.json",
```

Comment:
You notice we are ignoring the pandas datasource and it's be deleted from the repo because the new `SASAsset` schema cannot be generated.

I could also temporarily blacklist the `SASAsset` and add this back but... the `PandasDatasource.json` is now  almost 2k lines, due to it containing all the asset sub definitions. We could tweak the schema generation so that it uses external `$ref`s instead of internal ones but that's a bit of rabbit hole that doesn't really impact our launch.

Reply:
What's the issue with the `SASAsset`? Can we add a comment here as well? Same question for `Sqlite`.

Reply:
For sqlite I think it's the `SqliteDsn`, pydantic doesn't know what type to map it to. Which is a bit surprising, I would have just assumed it would treat it the same as its parent `AnyUrl` type.
https://docs.pydantic.dev/usage/schema/#json-schema-types
I'd need to look more into the pydantic internals.
https://docs.pydantic.dev/usage/types/#custom-data-types

For `SASAsset` it's something, similar, there's a field type somewhere that pydantic doesn't know how to map to a JSONSchema type. I'll update this comment with specifics once. If it's a simple fix I'll make it, if it requires much special handling I'll likely save it for a follow-up PR.